eng/pipelines/runtime.yml

```diff
@@ -385,6 +385,24 @@ jobs:
     jobParameters:
       liveRuntimeBuildConfig: release
 
+#
+# Build libraries using live CoreLib
+# These are part of the test coreclr generic test build.
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
+    buildConfig: Release
+    platforms:
+    - ${{ if ne(variables['debugOnPrReleaseOnRolling'], 'Release') }}:
```

Comment:
Instead use, `variables['isFullMatrix']` -> true, means Rolling build, false means PR.

Reply:
Interesting...  I used this incantation because of line 377 above.  I can switch, but this was obvious to me.